src/main/java/org/cactoos/list/Immutable.java

```diff
@@ -55,10 +56,18 @@
 
     /**
      * Ctor.
-     * @param src Source
+     * @param src Source collection
+     */
+    public Immutable(final Collection<T> src) {
+        this(new ListOf<>(src));
```

Comment:
@iakunin 
If you look at the issues [1220](https://github.com/yegor256/cactoos/issues/1220) and [1221](https://github.com/yegor256/cactoos/issues/1221) you could see that all the `Collections.unmodifiableCollection` and `Collections.unmodifiableList` should be replaced by 'Immutable'.
Your are using here `ListOf`, but `ListOf` has `Collections.unmodifiableList` which will be replaced by `Immutable`.
So, when it happens there will be circular dependency like that: 
![image](https://user-images.githubusercontent.com/3167940/68537911-ddf40f00-0339-11ea-84d9-d68df0ff6173.png)

Reply:
@fanifieiev that's a good point, thank you. Now nested list is copying by hand without using `ListOf`.